app/services/quotes_service.rb

```diff
@@ -0,0 +1,11 @@
+class QuotesService
+  def self.conn
+    Faraday.new(url: 'https://api.quotable.io/')
```

Comment:
I would remove the last `/` from the url. That will actually be part of the `conn.get` url.